libraries/exoplayer_rtsp/src/main/java/androidx/media3/exoplayer/rtsp/reader/RtpH263Reader.java

```diff
@@ -168,7 +170,7 @@ public void seek(long nextRtpTimestamp, long timeUs) {
   private void getBufferFlagsAndResolutionFromVop(ParsableByteArray data, boolean gotResolution) {
     // Picture Segment Packets (RFC4629 Section 6.1).
     // Search for SHORT_VIDEO_START_MARKER (0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0).
-    int currPosition = data.getPosition();
+    int currDataOffset = data.getPosition();
     long shortHeader = data.readUnsignedInt();
```

Comment:
what is the reason to name this `shortHeader`? It's not of type `short` and I can't find reference to any 'short' in the RFC

Reply:
Its taken from h263 decoder. Added link for the same in javaDoc.